shop/src/pages/checkout/Payment.js

```diff
@@ -13,6 +14,7 @@ import BetaWarning from './_BetaWarning'
 const CheckoutPayment = () => {
   const { config } = useConfig()
   const [{ cart }] = useStateValue()
+  usePGP()
```

Comment:
@nick Unrelated to this branch, but I had to add this for the checkout to work after merging with master (also in `themes/shared/checkout/Payment.js`)